addons/event/models/event_ticket.py

```diff
@@ -80,6 +80,8 @@ def default_get(self, fields):
     seats_used = fields.Integer(string='Used Seats', compute='_compute_seats', store=False)
     is_sold_out = fields.Boolean(
         'Sold Out', compute='_compute_is_sold_out', help='Whether seats are not available for this ticket.')
+    # reports
+    color = fields.Char('Color', default="#875A7B")
```

Comment:
Real bikeshedding: Use color of the community version
```suggestion
    color = fields.Char('Color', default="#71639E")
```

Reply:
That was partly a joke, but then you could change the default value in the `event_enterprise` module to match the enterprise theme.

Reply:
I get it :facepalm: :zzz: , I would rather go with previous color then, as we used a hardcoded background image with that color - not in community version theme - until last stable version. That way we keep something close to what we had. Also, it avoids creating a file just for setting that default color :p